Tests/test_file_dds.py

```diff
@@ -316,13 +347,35 @@ def test_save_unsupported_mode(tmp_path):
         im.save(out)
 
 
+@pytest.mark.parametrize(
+    ("mode", "test_file"),
+    (
+        ("L", "Tests/images/mode-l.dds"),
+        ("LA", "Tests/images/mode-la.dds"),
+        ("RGB", "Tests/images/mode-rgb.dds"),
+        ("RGBA", "Tests/images/mode-rgba.dds"),
+    ),
+)
+def test_open(mode, test_file):
+    with Image.open(test_file) as im:
+        assert im.mode == mode
+        assert_image_equal_tofile(im, test_file.replace(".dds", ".png"))
+
+
+def test_open_rgb8():
+    with Image.open("Tests/images/rgb8.dds") as im:
+        assert im.mode == "L"
+        assert_image_equal_tofile(im, "Tests/images/mode-l.png")
+
+
 @pytest.mark.parametrize(
     ("mode", "test_file"),
     [
         ("L", "Tests/images/linear_gradient.png"),
         ("LA", "Tests/images/uncompressed_la.png"),
         ("RGB", "Tests/images/hopper.png"),
         ("RGBA", "Tests/images/pil123rgba.png"),
+        ("L", TEST_FILE_ATI1),
```

Comment:
Did you have any particular reason for adding `test_open`, and for adding "L" to `test_save`? These tests don't fail without this PR, and it's not immediately apparent to me that it adds more coverage?

Reply:
for test_save, I probably wanted to test L saving and didn't notice that L was already tested

Reply:
about test_open, I don't really remember, but i think coverage tool was complaining about some path not being covered

Reply:
Hmm. That shouldn't be the case. For L, we have uncompressed_l.dds, for LA uncompressed_la.dds, for RGB hopper.dds and for RGBA uncompressed_rgb.dds.

I've created https://github.com/REDxEYE/Pillow/pull/9. See whether or not you'd like to merge. If I'm wrong, and there is missing coverage, it can be undone in some way.

Reply:
Thanks. The PR has been merged, and coverage is fine.